.github/labeler.yml

```diff
@@ -1,2 +1,32 @@
-maintenance:
-- any-glob-to-any-file: ['.github/**/*', '.pre-commit-config.yaml', 'VERSION']
+# This file is required by the the action 'https://github.com/actions/labeler'
+# and used in the '.github/workflows/ci_cd_pr.yml' workflow
+
+# -- Labels based on PR title ------------------------------------------------
+'fix':
+  - head-branch: ['fix']
+
+'enhancement':
+  - head-branch: ['feat']
+
+# -- Documentation labels ----------------------------------------------------
+'docs':
+  - all:
+      - changed-files:
+          - any-glob-to-any-file: ['README.rst', 'doc/source/**/*.rst']
+          - all-globs-to-all-files: ['!doc/styles/**', '!doc/.vale.ini']
+
+## -- Other labels ------------------------------------------------------------
+'code-style':
+  - any:
+      - changed-files:
+          - any-glob-to-any-file: ['.pre-commit-config.yaml', 'doc/.vale.ini']
+
+'ci':
+  - any:
+      - changed-files:
+          - any-glob-to-any-file: ['.github/workflows/**']
```

Comment:
what about dependencies label for all the changes in requirements folder? especially for dependabots.

Reply:
Good point! That´s configured via the `.github/workflows.yml` file. Just updated that in https://github.com/ansys/actions/pull/580/commits/c1bb01399e91015c2a0e52d27e7e603e731a1043